config/manifests/kustomization.yaml

```diff
@@ -6,3 +6,19 @@ resources:
 - ../metallb_rbac
 - ../samples
 - ../scorecard
+
+# These patches remove the unnecessary "cert" volume and its manager container volumeMount.
+patchesJson6902:
```

Comment:
Can you expand the need of removing the volumeMounts instead of avoiding creating them in the first place?